keyvi/include/keyvi/dictionary/fsa/generator.h

```diff
@@ -295,7 +290,12 @@ class Generator final {
     }
 
     stream << KEYVI_FILE_MAGIC;
-    WriteHeader(stream);
+
+    keyvi::dictionary::DictionaryProperties p(2, start_state_, number_of_keys_added_, number_of_states_,
```

Comment:
The `2` here refers to the version. I think it makes sense to create a constant somewhere with the current version.